cg/apps/housekeeper/hk.py

```diff
@@ -110,6 +110,7 @@ def add_file(
         )
 
         new_file.version: Version = version_obj
```

Comment:
This used to rely on the `version_obj` being in the session and `new_file` being merged into the session as well.